src/managers/ApplicationCommandManager.js

```diff
@@ -115,14 +115,14 @@ class ApplicationCommandManager extends CachedManager {
    */
   async create(command, guildId) {
     const data = await this.commandPath({ guildId }).post({
-      data: this.constructor.transformCommand(command),
+      data: ApplicationCommand.isAPICommandData(command) ? command : this.constructor.transformCommand(command),
```

Comment:
Instead of changing this everywhere, you should use the transformCommand function and just do
```js
default_permission: command.defaultPermission ?? comand.default_permission
```